src/main/kotlin/lotto/domain/Rank.kt

```diff
@@ -1,17 +1,18 @@
 package lotto.domain
 
-enum class Rank(val hitCount: Int, val prize: Long) {
-    LastPlace(0, 0),
-    FourthPlace(3, 5_000),
-    ThirdPlace(4, 50_000),
-    SecondPlace(5, 1_500_000),
-    FirstPlace(6, 2_000_000_000);
+enum class Rank(val hitCount: Int, val needBonusHit: Boolean, val prize: Long) {
+    FirstPlace(6, false, 2_000_000_000),
+    SecondPlace(5, true, 3_000_000),
```

Comment:
```suggestion
    SecondPlace(5, needBonusHit = true, 3_000_000),
```
needBonusHit는 default Value를 사용해보면 어떨까요? true일때만 명시해도 좋을거같아요!